picard/ui/metadatabox.py

```diff
@@ -309,16 +309,28 @@ def contextMenuEvent(self, event):
             if len(tags) == 1 or removals or useorigs:
                 menu.addSeparator()
             menu.addAction(self.add_tag_action)
+            add_to_preserved_tags_action = QtGui.QAction(_(u"Add Selected Tags to 'Preserve Tags' List"), self.parent)
```

Comment:
Thinking about this some more (again), I think you can drop "Selected Tags" here because neither "Edit" nor "Remove" have it.